tmpl/header.tmpl

```diff
@@ -0,0 +1,43 @@
+{{ define "header" }}
+    <head>
+        <meta name="viewport" content="width=device-width" />
+        <meta charset="utf-8" />
+        <title>Verify HWSC Account</title>
+        <style type="text/css">
+            body {
+                margin: 0 auto;
+                padding: 0;
+            }
+            .small-print {
+                font-size: 12px;
+                color: #656565;
+            }
+            .button-container {
+                padding: 20px 0 40px 0;
```

Comment:
can we use relative units?

Reply:
any discussion?

Reply:
I did use relative units...

Reply:
https://www.tutorialrepublic.com/css-tutorial/css-units.php